images/cert-manager/tests/main.tf

```diff
@@ -0,0 +1,78 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digests" {
+  description = "The image digests to run tests over."
+  type = object({
+    acmesolver = string
+    cainjector = string
+    controller = string
+    webhook    = string
+  })
+}
+
+variable "suffix" {
+  description = "Suffix to append to the helm release name."
+  default     = ""
+}
+
+variable "skip_crds" {
+  default = false
+}
+
+resource "helm_release" "cert-manager" {
+  name             = "cert-manager${var.suffix}"
```

Comment:
Can we make this generate a random suffix instead (see what Jason did in his PR)?